torchvision/transforms.py

```diff
@@ -27,6 +27,13 @@ def __call__(self, img):
             img = t(img)
         return img
 
+class FilenameToPILImage(object):
```

Comment:
All other datasets handle the image loading internally making the use of this transform not so generic. There is an argument for changing the other datasets to use a transform for this, not sure what the original motivation for not doing this was cc @fmassa who might know more